src/App.vue

```diff
@@ -95,29 +95,63 @@
             </div>
         </nav>
 
-        <class-view v-if="sideBar.showSelectClass"></class-view>
+        <class-view
+            v-if="sideBar.showSelectClass"
+            class="bg-light"
+            :class="'sidebar-nocol' + classMobile"
+        ></class-view>
 
-        <fuzzy-view v-else-if="sideBar.showFuzzy"> </fuzzy-view>
-
-        <event-view v-else-if="sideBar.showEvent"> </event-view>
-
-        <filter-view v-else-if="sideBar.showFilter"></filter-view>
-
-        <display-view v-else-if="sideBar.showSetting"></display-view>
-
-        <export-view v-else-if="sideBar.showExport"></export-view>
+        <fuzzy-view
+            v-else-if="sideBar.showFuzzy"
+            class="bg-light"
+            :class="'sidebar-nocol' + classMobile"
+        >
+        </fuzzy-view>
 
-        <palette-view v-else-if="sideBar.showSelectColor"></palette-view>
+        <event-view
+            v-else-if="sideBar.showEvent"
+            class="bg-light"
+            :class="'sidebar-nocol' + classMobile"
+        >
+        </event-view>
+
+        <filter-view
+            v-else-if="sideBar.showFilter"
+            class="bg-light"
+            :class="'sidebar-nocol' + classMobile"
```

Comment:
Don't use `sidebar-nocol` everywhere. It is a replacement of the `sidebar` class when you require the background of some `list-group-item` to be pure white. If it is not required, please do not use it.